users/user-guide-query/query-options.md

```diff
@@ -57,6 +57,16 @@ description: This document contains all the available query options
         See <a data-mention href="grouping-algorithm.md#configuration-parameters">#configuration-parameters</a></td>
       <td>Server level config</td>
     </tr>
+    <tr>
+       <td><strong>skipIndexes</strong></td>
+       <td>
+         <p>Which indexes to skip usage of (i.e. scan instead), per-column.  This is useful for side-by-side comparison/debugging.  There can be cases where the use of an index is actually more expensive than performing a scan of the docs which match other filters.  One such example could be a low-selectivity inverted index used in conjunction with another highly selective filter.</p>
+         <p>Config can be specified using url parameter format: <code>skipIndexes='col1=inverted,range&col2=inverted'</code></p>
```

Comment:
It would be helpful if we can list down all indexes that can be skipped

Reply:
Agreed!  Added in https://github.com/pinot-contrib/pinot-docs/pull/340/commits/88f30535973f86a453e5cb0e879d2e2e53e1fe00